src/java.base/share/classes/jdk/internal/util/StaticProperty.java

```diff
@@ -98,19 +98,33 @@ private StaticProperty() {}
         USER_LANGUAGE = getProperty(props, "user.language", "en");
         USER_LANGUAGE_DISPLAY = getProperty(props, "user.language.display", USER_LANGUAGE);
         USER_LANGUAGE_FORMAT = getProperty(props, "user.language.format", USER_LANGUAGE);
-        USER_SCRIPT = getProperty(props, "user.script", "");
+        // for compatibility, check for old user.region property
+        USER_REGION = getProperty(props, "user.region", "");
+        if (!USER_REGION.isEmpty()) {
+            // region can be of form country, country_variant, or _variant
```

Comment:
typo "of form"? "region can be from"...

Reply:
I am not the author of this comment, but I think they meant it, like "one of forms".